spec/system/admin/reports/orders_and_fulfillment_spec.rb

```diff
@@ -438,5 +440,167 @@
         end
       end
     end
+
+    describe "Saving report rendering options" do
+      let(:report_title) do
+        "Order Cycle Supplier Totals"
+      end
```

Comment:
This code is fine, but FYI the more compact syntax can be used:

```suggestion
      let(:report_title) { "Order Cycle Supplier Totals" }
```